docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       - BTC_TXINDEX=1
       - BTC_PRUNE=0
       - BTC_DISABLEWALLET=1
+      - BTC_RPCALLOWIP=172.33.0.0/16
```

Comment:
Gonna need more context about why you want this implemented since AFAIK this will not work for your desires.  Please create an issue with details and link it to this PR.  However I don't think this will help, and regardless will need to be undone during a coming refactor with internal IP addresses and we are moving off of the subnet you added since its a public IP block.  Please make an issue https://github.com/dappnode/DAppNodePackage-bitcoin/issues and explain in more detail your issue because just allowing the ip range wont help since you need to have an exposed port to the host. not to the internal docker network.